packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md

```diff
@@ -340,45 +284,43 @@ like the following:
 
 </div>
 
-<div class="padded-line"></div>
-
-> **Note:** okta-dac is not an official Okta product and does not
+> **Note:** okta-dac isn't an official Okta product and doesn't
 qualify for any support. It's recommended to work with your preferred Okta
 Solution Provider to integrate this code within your existing portal.
 
 #### More information
 
-For more information about okta-dac visit
-[here](https://docs.idp.rocks/).
+See [okta-dac](https://docs.idp.rocks/) for more information.
 
 ## Configuration 2
 
-<div class="heading-2-subtext">Host tenants in separate orgs (e.g. Hub and Spoke)</div>
+<div class="heading-2-subtext">Host tenants in separate orgs (for example,
+Hub and Spoke)</div>
 
 ### Summary
 
-This configuration separates tenants into separate containers (aka orgs).
+This configuration separates tenants into separate containers (orgs).
 The most popular architecture for this configuration is known as Hub and Spoke.
 
 ### When to use this configuration
 
 Reasons why organizations choose this configuration include:
 
 * **Data residency requirements:** The data in tenants can more easily be
-isolated when placed into separate orgs.  This isolation becomes relevant
-when user data is required to reside only in the user’s country (e.g.
-government regulations).
-* **Strong delegated admin support:**  Although there is delegated admin
+isolated when placed into separate orgs. This isolation becomes relevant
+when user data is required to reside only in the user’s country (for
+example, government regulations).
+* **Strong delegated admin support:** Although there is delegated admin
 support in [okta-dac](#the-okta-dac-project), the amount of supported delegated
```

Comment:
I really struggled with this sentence - how about: 
```suggestion
support in [okta-dac](#the-okta-dac-project), the number of delegated
```

Reply:
Updated in the latest commit.